packages/ckeditor5-table/tests/tableproperties/tablepropertiesui.js

```diff
@@ -707,6 +721,15 @@ describe( 'table properties', () => {
 				sinon.assert.calledOnce( spy );
 			} );
 
+			it( 'should not reposition the baloon if view is not visible', () => {
+				const spy = sinon.spy( contextualBalloon, 'updatePosition' );
+
+				tablePropertiesUI.view = false;
+				editor.ui.fire( 'update' );
+
+				expect( spy.called ).to.be.false;
+			} );
```

Comment:
What's up with this test? :)

Reply:
That's a coverage test. It seems that after removing code that was not connected to that case, the coverage've dropped. I've checked manually whether it should be covered and while I'm not sure because I couldn't manually get to that path, I still added some coverage test to not create a reggrestion.

Reply:
I see that there are already similar tests there (checking repositioning), so OK.